backend/src/main/java/com/htecgroup/skynest/security/SecurityConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.htecgroup.skynest.security;
+
+public class SecurityConstants {
+
+    private SecurityConstants(){
+        throw new IllegalStateException("Utility class");
+    }
+    public static final long EXPIRATION_TIME = 180000; // 30 minutes
+    public static final String TOKEN_PREFIX = "Bearer ";
+    public static final String HEADER_STRING = "Authorization";
+    public static final String SING_UP_URL = "/users/register";
```

Comment:
just a spelling error, should be `SIGN_UP_URL`

Reply:
Nice catch, I'll change it.

Reply:
that would be a nice functionality - SING UP :D